Update UserGuide.md (#197)
AY2223S1-CS2103-F14-3/tp: docs/UserGuide.md

The data of all the internship applications is saved automatically to the hard disk whenever any changes are made, so no manual saving is necessary.

<div markdown="block" class="alert alert-danger">

**:information_source: Notes about the CinternS save file:**<br>

* A new folder `data` will be created with the file `applicationbook.json`. This is where your data will be stored for the next time you use the app, transferring it to another device or sharing with your friend.

> * Please refrain from editing this save file as the file might become corrupted. When CinternS is booted, if the file is corrupted, it will reset the save file. You will NOT be able to retrieve your data!

Reviewer: Maybe we can make this line less harsh, especially since the ability to edit the save file is technically a requirement. How about something like this?

```suggestion
* Editing the save file directly is discouraged as it might become corrupted. When CinternS is booted, if the file is corrupted, it will reset the save file. You will NOT be able to retrieve your data!
```

* If you do wish to edit the save file, please make sure every field is included as specified in the above commands (e.g. each application and interview must have all the fields listed as required above). Do make a copy of your save file before making any edits in the case that your data is lost so that you can recover it (by replacing the save file with the copy).

> 

Reviewer 2: In addition to what @teekaytai said, perhaps can also add something to tell user what to do when the file is corrupted and they cannot correct it back.
```suggestion
* If the save file is corrupted due to your changes and you cannot find the way to correct it back, the last resort is to delete the save file and reboot CinternS. CinternS will create a new save file for you. Do note that your still can't retrieve your previous data back.
```

| Action          | Format, Examples                                                                                                                                                                                               |
|-----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| **Help**        | `help`                                                                                                                                                                                                         |

> | **Add**         | `add c/COMPANY ct/CONTACT d/DATE_APPLIED e/EMAIL p/POSITION s/STATUS [t/TAG]...` <br> e.g., `add c/Google ct/60221023 d/2022-01-01 e/google@example.com p/Software Engineer s/pending t/preferred` |

Reviewer: Remember to reorder these parameters too, thanks!
